### ISSUE TITLE
[Editor] Add a conditional breakpoint pane close button

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#1945]
When we open the conditional breakpoint pane we should show a close button on the far right.

This is similar to the editor search bar and source search bar, which both have a close button.

Here's a patch to help you get started
```diff
diff --git a/src/components/Editor/ConditionalPanel.js b/src/components/Editor/ConditionalPanel.js
index 385f934..f66fda1 100644
--- a/src/components/Editor/ConditionalPanel.js
+++ b/src/components/Editor/ConditionalPanel.js
@@ -26,6 +26,11 @@ function renderConditionalPanel({ condition, closePanel, setBreakpoint }:
         defaultValue: condition,
         placeholder: L10N.getStr("editor.conditionalPanel.placeholder"),
         onKeyPress: onKey
+      }),
+
+      CloseButton({
+        handleClick: closePanel,
+        buttonClass: "big"
       })
     ),
     panel
```

### Screenshot

![conditional breakpoints](https://cloud.githubusercontent.com/assets/254562/22650913/159c6dbc-ec4f-11e6-8048-21874611809a.png)

**Comment by aklt** [comment]
I'm having a look at this ;-)

**Comment by ryanjduffy** [comment]
I had started on this as well yesterday but @aklt showed better form by actually commenting here! I'll share what I have to possibly accelerate things a bit. I took things a bit further and altered the UX such that:

* Using the 'x' saves and closes the panel
* Pressing "Escape" closes without saving
* Pressing "Enter" always saves. If there is no content, it converts to a "regular" breakpoint

Open for discussion, of course.

https://github.com/ryanjduffy/debugger.html/tree/feature/1945

![closebutton](https://cloud.githubusercontent.com/assets/788456/23366753/82979638-fccd-11e6-991d-fa0d612f1014.gif)
